ci/gcov/coverage-generator.sh

```diff
@@ -38,7 +38,7 @@ check_dependency rpm2cpio
 check_dependency cpio
 
 # Build unit test coverage rpm
```

Comment:
Please update.  for readability and maintenance.

`# Build a RPM file to geneate a code coverage statistics with gcov/lcov.`

Reply:
fixed